package.json

```diff
@@ -4,12 +4,18 @@
   "description": "WordPress Theme framework",
   "main": "index.js",
   "scripts": {
+	"bs": "browser-sync start --config build/browser-sync.config.js",
     "css": "npm-run-all css-compile css-postcss css-minify",
     "css-compile": "sass --style expanded --source-map --embed-sources --no-error-css sass/theme.scss:css/theme.css sass/custom-editor-style.scss:css/custom-editor-style.css",
     "css-minify": "cleancss -O1 --format breakWith=lf --with-rebase --source-map --source-map-inline-sources --output css/ --batch --batch-suffix \".min\" \"css/*.css\" \"!css/*.min.css\" \"!css/*rtl*.css\"",
-    "css-postcss": "postcss --config postcss.config.js --replace \"css/*.css\" \"!css/*.rtl*.css\" \"!css/*.min.css\"",
-    "watch": "npm-run-all --parallel watch-*",
-    "watch-css": "nodemon --watch sass/ --ext scss --exec \"npm-run-all css\""
+    "css-postcss": "postcss --config build/postcss.config.js --replace \"css/*.css\" \"!css/*.rtl*.css\" \"!css/*.min.css\"",
+    "js": "npm-run-all js-compile js-minify",
+    "js-compile": "rollup --config build/rollup.config.js --sourcemap",
+    "js-minify": "terser --compress typeofs=false --mangle --comments \"/^!/\" --source-map \"content=js/theme.js.map,includeSources,url=theme.min.js.map\" --output js/theme.min.js js/theme.js",
```

Comment:
should probably add a terser config file or options to customize terser options.
ie. CI/CD might only be able to run on a single thread where i might want to run it on 4

Reply:
Can you send example? I didn't see one in Bootstrap so I'm not sure what it would look lik

Reply:
You see how they are calling terser and a series of flags
```
terser --compress typeofs=false --mangle --comments \"/^!/\" --source-map \"content=js/theme.js.map,includeSources,url=theme.min.js.map\" --output js/theme.min.js js/theme.js
```
That is not a very sustainable way of doing it for instance if I want to toggle `parallel` based on a specific scenario dev/prod build.

Here is an example from a project i got with webpack
```
            minimizer: [
                (compiler) => {
                    const TerserPlugin = require('terser-webpack-plugin');
                    new TerserPlugin({
                        parallel: true,
                        test: /\.js($|\?)/i,
                        // sourceMap: IS_DEV,
                        terserOptions: {
                            compress: {
                                booleans: IS_PROD,
                                conditionals: IS_PROD,
                                drop_console: IS_PROD,
                                drop_debugger: IS_PROD,
                                if_return: IS_PROD,
                                join_vars: IS_PROD,
                                keep_classnames: IS_DEV,
                                keep_fnames: IS_DEV,
                                reduce_vars: IS_PROD,
                                sequences: IS_PROD,
                                warnings: IS_DEV,
                                ecma: 5,

                            },
                            output: {
                                comments: IS_DEV,
                            },
                        },
                    }).apply(compiler);
                },
                new OptimizeCSSAssetsPlugin({}),
            ],
```

You can do something similar with `rollup-plugin-terser`

Reply:
👍 New issue to track this here: https://github.com/understrap/understrap/issues/1400